test/utils/PenaltyDetailsConverterSpec.scala

```diff
@@ -24,123 +24,133 @@ import models.hipPenaltyDetails.breathingSpace.BreathingSpace
 import models.hipPenaltyDetails.latePayment._
 import models.hipPenaltyDetails.lateSubmission._
 import models.hipPenaltyDetails.{Totalisations, PenaltyDetails => HIPPenaltyDetails}
+import utils.PenaltyDetailsConverter.putSeqInsideOption
 
 import java.time.{Instant, LocalDate}
 
 class PenaltyDetailsConverterSpec extends SpecBase {
 
   "PenaltyDetailsConverter" should {
-    
+
```

Comment:
The only non automatic formatter changes are a the 2 new tests at the end of this file